src/Transporters/Http.php

```diff
@@ -35,7 +35,12 @@ public function request(string $method, array $params = []): array|string|bool
             'params'  => $params,
         ]);
 
-        $request = new Request('POST', $this->url, [], $body);
+        $headers = [
+            'Content-Type'   => 'application/json',
+            'Content-Length' => strlen($body),
```

Comment:
Is the `Content-Length` required? 🤔

Reply:
`Content-Length ` is not required, just make request headers more standard.

Reply:
So, everyone generic Ethereum RPC, works with this headers?

Reply:
Fair 👍🏻 I thought this was applied by PSR-7 compatible clients (such as Guzzle) automatically, but apparently not.

All Ethereum APIs are JSON-RPC, so this should in theory be supported by all of them. 🤔